File: Sirma Academy May 2024/JavaScript/Conditional Statements/AdventureGamePathDecision.js
function adventureGame(itemOne, itemTwo) {
  let pathToTake = "";

  if (itemOne == "sword" || itemTwo == "sword") {
    if (itemOne == "shield" || itemTwo == "shield") {
      pathToTake = "Path to the castle";
    } else {
      pathToTake = "Path to the forest";
    }
  }
  if (itemOne == "map" || itemTwo == "map") {
    if (itemOne == "coins" || itemTwo == "coins") {
      pathToTake = "Go to the town";
    } else {
      pathToTake = "Camp at the current spot and prepare for the next day.";
    }
  }
  if (!pathToTake) {
    pathToTake = "Wander aimlessly";
  }

  console.log(pathToTake);
}

adventureGame("map", "coins");

File: Sirma Academy May 2024/Java/Conditional Statements/src/AdventureGamePathDecision.java
import java.util.Scanner;

public class AdventureGamePathDecision {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String itemOne = sc.nextLine();
        String itemTwo = sc.nextLine();
        String pathToTake = "";

        if(itemOne.equals("sword") || itemTwo.equals("sword")){
            if(itemOne.equals("shield") || itemTwo.equals("shield")){
                pathToTake = "Path to the castle";
            } else {
                pathToTake = "Path to the forest";
            }
        }
        if(itemOne.equals("map") || itemTwo.equals("map")){
            if(itemOne.equals("coins") || itemTwo.equals("coins")){
                pathToTake = "Go to the town";
            } else {
                pathToTake = "Camp at the current spot and prepare for the next day.";
            }
        }
        if(pathToTake.isEmpty()){
            pathToTake = "Wander aimlessly";
        }
        System.out.println(pathToTake);
    }
}
